_includes/charts/tigera-operator/templates/tigera-operator/02-configmap-calicoctl-resources.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if eq .Values.kubernetesProvider "openshift" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: calicoctl-resources
+  namespace: tigera-operator
+data:
+  # To have resources created before Calico resources are created add an
```

Comment:
I think this is a bit confusing - "To have resources created before Calico resources are created".